app/src/components/missions/application/MissionApplicationTabs.tsx

```diff
@@ -174,7 +174,13 @@ export function MissionApplicationTabs({
                   </p>
 
                   <div className="flex gap-4">
-                    <Button className="w-44" variant={"destructive"}>
+                    <Button
+                      className="w-44"
+                      variant={"destructive"}
+                      onClick={() => {
+                        router.push(`/projects/new`)
+                      }}
+                    >
```

Comment:
Please use [ExtendedLink](https://github.com/voteagora/op-atlas/blob/a02f1e2403641d83397dcaac1df9fdc14f28151d/app/src/components/common/ExtendedLink.tsx) component instead of pushing via router.

```jsx
<ExtendedLink as="button" text={"Add Project"} href='/projects/new' />
```